cmd/deploy/remote.go

```diff
@@ -224,3 +226,19 @@ func getOriginalCWD(cwd, manifestPath string) string {
 	manifestPathDir := filepath.Dir(fmt.Sprintf("/%s", manifestPath))
 	return strings.TrimSuffix(cwd, manifestPathDir)
 }
+
+func getOktetoCLIVersion() string {
```

Comment:
at some point we should unify shared code between deploy/destroy remote commands